app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -396,6 +401,22 @@ class BrowserTabFragment :
         removeDaxDialogFromActivity()
         renderer = BrowserTabFragmentRenderer()
         decorator = BrowserTabFragmentDecorator()
+        voiceSearchLauncher.registerResultsCallback(this, requireActivity(), BROWSER) {
```

Comment:
Maybe only register this callback if voice search is available?

Reply:
I initially wanted to do that but the ActivityResultRegistry.register() which is used in the implementation needs to happen on the Activity's onCreate() so I can't group it together when we get the OminbarViewState emission.